Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -249,11 +282,36 @@ int main(int argc, char** argv)
 {
     auto app = GUI::Application::construct(argc, argv);
 
-    if (pledge("stdio thread recvfd sendfd rpath", nullptr) < 0) {
+    if (pledge("stdio thread recvfd sendfd rpath unix", nullptr) < 0) {
         perror("pledge");
         return 1;
     }
 
+    if (unveil("/tmp/portal/filesystemaccess", "rw") < 0) {
+        perror("unveil");
+        return 1;
+    }
+
+    if (unveil("/home/anon/Documents/3D Models/teapot.obj", "r") < 0) {
```

Comment:
Seems weird to unveil these explicitly if we're using FSAC.

Reply:
This is absolutely fine for sandboxing. It is nice to reduce the amount of unveils but these are demo files which are opened every time the application is opened.